README.md

```diff
@@ -94,49 +92,46 @@ Now, go back to the development branch:
 git checkout develop
 ```
 
-### 4. Configure the AWS ECR repository
-
-Create a docker repository for your plugin in the `hyp3-v2-full-access` account:
-   ```bash
-   # Assuming your aws cli is setup to use the hyp3-v2-full-access profile
-   aws ecr create-repository \
-       --repository-name hyp3-<process> \
-       --image-scanning-configuration scanOnPush=true \
-       --region us-west-2
-   ```
+### 4. Configure the GitHub repository settings
```

Comment:
Might need to add a section on configuring GHCR. One point to make:
* "packages" (here, container images) are always private by default, so they'll need to be made public explicitly after the first image is pused.

Reply:
I could use some help fleshing out a section on the GHCR. Not sure I understand how it works!

Reply:
Let's actually punt talking about GHCR until we use this for the hyp3-isce plugin, so we know exactly what steps we'll need to take